### PR TITLE
Adds config options to explicitly initialize adapter state. Add after…

### PR DESCRIPTION
**Pull request by KapitonovRoman**
… initialize rails hook to setup adapters. Fixes rubocop warnings. Removes topic from the list of google cloud adapter arguments.

### CODE REVIEW

File: Gemfile.lock [hunk 1]
```diff
@@ -110,7 +110,7 @@ GEM
       multi_json (~> 1.11)
       os (>= 0.9, < 2.0)
       signet (~> 0.12)
-    grpc (1.25.0)
+    grpc (1.25.0-universal-darwin)
```
Comment (mbhnyc) [inline Gemfile.lock]: does this change match the other gems?

Comment (KapitonovRoman) [reply Gemfile.lock]: whoooooooops

File: lib/stenotype/railtie.rb [hunk 2]
```diff
@@ -18,6 +18,8 @@ class Railtie < ::Rails::Railtie
 
     config.stenotype = Stenotype.config
 
+    config.after_initialize { config.stenotype.targets.each(&:setup!) } if config.stenotype.rails.explicit_client_initialization
```
Comment (mbhnyc) [inline lib/stenotype/railtie.rb]: what about "auto_initialize" as a name for this option?

File: README.md [hunk 3]
```diff
@@ -66,6 +67,10 @@ Specifies a logger for messages and exceptions to be output to. If not set defau
 
 This flag if set to `true` is going to suppress all `StandardError`'s raised within a gem. Raises the error to the caller if set to `false`
 
+#### config.explicit_client_initialization
+
+Allows to enable/disable explicitly setup the adapter. For example client and topic for google cloud.
```
Comment (mbhnyc) [inline README.md]: change comments to match - the language or is confusing, if this option is off then you are explicitly setting up the adapter? the language here is confusing, if this option is off _then_ you are explicitly setting up the adapter? Right?

File: README.md [hunk 4]
```diff
@@ -31,10 +31,11 @@ Stenotype.configure do |config|
...
 
-  config.uuid_generator          = SecureRandom
-  config.dispatcher              = Stenotype::Dispatcher.new
-  config.logger                  = Logger.new(STDOUT)
-  config.graceful_error_handling = true
+  config.uuid_generator               = SecureRandom
+  config.dispatcher                   = Stenotype::Dispatcher.new
+  config.logger                       = Logger.new(STDOUT)
+  config.graceful_error_handling      = true
+  config.force_adapter_initialization = true
```
Comment (mbhnyc) [inline README.md]: we're getting closer but this is still weird :D 

the word "force" is generally a negative in English, so it makes it sound like a bad thing, when actually it is a good thing and the default :).

Let's just rename THIS option to `auto_adapter_initialization` and I think we have it.

**Pull request merged**